wiki/source/quilt.md

```diff
@@ -0,0 +1,52 @@
+---
+title: tilde quilt
+author: lab6
+category: tilde.club
+---
+
+## 2-dimensional anonymous collaborative microblogging
+
+[Visit the tilde quilt](https://tilde.club/~lab6/quilt.html) to read messages from other tilde users.
+
+Add your own messages (up to 150 printable ASCII characters) by running a command like the following:
+
+`echo "Hi!" > /home/lab6/quilt`
```

Comment:
did you mean to append to the file with `>>` instead?

Reply:
>  worked when I posted a message on it.. so I assume its designed around using > instead of >>

Reply:
deepend is right - the destination is a named pipe, not a file, so either `>` or `>>` will work.